server.ts

```diff
@@ -26,9 +26,33 @@ app.use(cors()) //add CORS support to each following route handler
 const client = new Client(dbConfig);
 client.connect();
 
-app.get("/", async (req, res) => {
-  const dbres = await client.query('select * from categories');
-  res.json(dbres.rows);
+
+// To get top10 most voted dogs and send them back in order from largest to smallest.
+app.get("/votes", async (req, res) => {
+  try {
+    const topTen = await client.query('select breed_name, count(breed_name) from votes group by breed_name order by count(breed_name) desc limit 10');
+    res.json(topTen.rows);
+  } catch (error) {
+    console.log(error);
```

Comment:
Try send res.status(error) back to the front end on the catch. And then send res.status(200) to the front end on the try :)